src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/model/UpdatePaymentMessage.java

```diff
@@ -6,20 +6,17 @@
 
     public final String envelopeId;
     public final String jurisdiction;
-    public final String service;
     public final String exceptionRecordRef;
     public final String newCaseRef;
 
     public UpdatePaymentMessage(
-        @JsonProperty(value = "envelope_id", required = true) String envelopeId,
-        @JsonProperty(value = "jurisdiction", required = true) String jurisdiction,
-        @JsonProperty(value = "service", required = true) String service,
+        @JsonProperty(value = "envelope_id") String envelopeId,
+        @JsonProperty(value = "jurisdiction") String jurisdiction,
```

Comment:
Are these two fields not required anymore?

Reply:
@lgonczar only `service` was removed. It was only used for logging

Reply:
But now `envelope_id` and `jurisdiction` are not required. Was it intended?

Reply:
thought without envelope id and jurisdiction we should process the payment.  so made them optional